### ISSUE TITLE
A lot of novels don't have chapters in then all are missing!! 

### ISSUE THREAD

**Issue by Samrandomreader11** [MakeYourLifeEasier/novel-ebook.com#259]
**Describe the bug**
I thought it was just the fanfic that is by web novel that have their chapters missing but when I download the master of the end of times and Daoist Master of Qing Xuan all their chapters are the same MISSING.....  Am I the only one having this problem? Please fix them please I am on this site for so long please fix this thank you

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by killerclown3** [comment]
i have the same issue with all novels i try to download

**Comment by s-techy** [comment]
Same here, tried to download subsequent chapters but has the same issue as above. novel: World defying Dan god

**Comment by NIGHT49** [comment]
Same issue with near a dozen books. The chapteres look fine when scrolling to select but once downloaded reads as chapters missing. (For 14 books and counting)
(Browser: forefox,explorer and chrome android and windows 10.)

**Comment by chaoss-west** [comment]
I also happen to have  something like this

**Comment by Elle-57** [comment]
Yeah me. I hope this problem can be fix. I have been in this site for a long time. Im a supporter. Thank you
